docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it might make a request to GitHub to get the latest tarball of the Minishift or OpenShift binary.
+Sometimes, during these requests, you might receive a 403 forbidden status from GitHub if your network reached the API rate limit and the command does not succeed, for example:
```

Comment:
I would change to:

"Sometimes, during these requests, you might receive a 403 forbidden status from GitHub if your IP address reached the API rate limit. In this case the command will fail and you will receive a message, for example:"